internal/pkg/scorecard/scorecard.go

```diff
@@ -99,277 +49,36 @@ var (
 	log           = logrus.New()
 )
 
-func runTests() ([]scapiv1alpha1.ScorecardOutput, error) {
-	defer func() {
-		if err := cleanupScorecard(); err != nil {
-			log.Errorf("Failed to cleanup resources: (%v)", err)
-		}
-	}()
+var rootDir string
```

Comment:
Any reason for this global? Why not call `os.Getwd()` where needed?

Reply:
I use this variable in the `Run` function for `internalPlugin`. Since the external plugins run in a different directory, I automatically `chdir` to the `rootDir` before running internal plugins to make sure I'm in the right place.